recipes/sqlite3/all/conanfile.py

```diff
@@ -80,24 +88,28 @@ def _configure_cmake(self):
         self._cmake.definitions["SQLITE3_BUILD_EXECUTABLE"] = self.options.build_executable
         self._cmake.definitions["THREADSAFE"] = self.options.threadsafe
         self._cmake.definitions["ENABLE_COLUMN_METADATA"] = self.options.enable_column_metadata
+        self._cmake.definitions["ENABLE_DBSTAT_VTAB"] = self.options.enable_dbstat_vtab
         self._cmake.definitions["ENABLE_EXPLAIN_COMMENTS"] = self.options.enable_explain_comments
         self._cmake.definitions["ENABLE_FTS3"] = self.options.enable_fts3
         self._cmake.definitions["ENABLE_FTS3_PARENTHESIS"] = self.options.enable_fts3_parenthesis
         self._cmake.definitions["ENABLE_FTS4"] = self.options.enable_fts4
         self._cmake.definitions["ENABLE_FTS5"] = self.options.enable_fts5
         self._cmake.definitions["ENABLE_JSON1"] = self.options.enable_json1
+        self._cmake.definitions["ENABLE_PREUPDATE_HOOK"] = self.options.enable_preupdate_hook
         self._cmake.definitions["ENABLE_SOUNDEX"] = self.options.enable_soundex
         self._cmake.definitions["ENABLE_RTREE"] = self.options.enable_rtree
+        self._cmake.definitions["ENABLE_UNLOCK_NOTIFY"] = self.options.enable_unlock_notify
+        self._cmake.definitions["ENABLE_DEFAULT_SECURE_DELETE"] = self.options.enable_default_secure_delete
         self._cmake.definitions["USE_ALLOCA"] = self.options.use_alloca
         self._cmake.definitions["OMIT_LOAD_EXTENSION"] = self.options.omit_load_extension
-        self._cmake.definitions["SQLITE_ENABLE_UNLOCK_NOTIFY"] = self.options.enable_unlock_notify
         self._cmake.definitions["HAVE_FDATASYNC"] = True
         self._cmake.definitions["HAVE_GMTIME_R"] = True
         self._cmake.definitions["HAVE_LOCALTIME_R"] = self.settings.os != "Windows"
         self._cmake.definitions["HAVE_POSIX_FALLOCATE"] = not (self.settings.os in ["Windows", "Android"] or tools.is_apple_os(self.settings.os))
         self._cmake.definitions["HAVE_STRERROR_R"] = True
         self._cmake.definitions["HAVE_USLEEP"] = True
         self._cmake.definitions["DISABLE_GETHOSTUUID"] = self.options.disable_gethostuuid
+        self._cmake.definitions["MAX_BLOB_SIZE"] = self.options.max_blob_size
```

Comment:
I can't find any reference about MAX_BLOB_SIZE in https://www.sqlite.org/doc_keyword_crossref.html. Where can find more information?

Reply:
SQLITE_MAX_LENGTH is the preprocessor macro. I don't find max_length to really be satisfactorily self-descriptive as an option, which is why I changed it. I can revert it to the less-descriptive max_length if we want to adhere more closely to the sqlite macros.